### PR TITLE
feat: support multiple css file extensions in development

### PR DESCRIPTION
**Pull request by NeonGC**

### CODE REVIEW

File: src/Configuration.php [hunk 1]
```diff
@@ -257,7 +257,13 @@ protected function createDevelopmentTag(string $path): string
...
         $url = Str::of($this->config('dev_server.url'))->finish('/')->append($path);
 
-        if (Str::endsWith($path, '.css')) {
+        if (Str::endsWith($path, '.css')
+            || Str::endsWith($path, '.scss')
+            || Str::endsWith($path, '.sass')
+            || Str::endsWith($path, '.less')
+            || Str::endsWith($path, '.styl')
+            || Str::endsWith($path, '.stylus')
+        ) {
```
Comment (innocenzi) [inline src/Configuration.php:266]:
```suggestion
        if (Str::endsWith($path, ['.css', '.scss', '.sass', '.less', '.styl', '.stylus']) {
```

Comment (innocenzi) [reply src/Configuration.php:266]: Can you describe exactly the scenario that happens when this check only looks for `.css`?

A new test would be ideal.

**Review by innocenzi — CHANGES_REQUESTED** [review]

**Comment by NeonGC** [comment]
If you had index.scss in resources/scripts and tried to do @vite('index.scss') it won't work on dev mode, but work on production mode

**Comment by innocenzi** [comment]
Sorry, my fix was missing a parenthesis. Can you fix that? Once all tests pass I can merge this.

**Review by innocenzi — APPROVED** [review]

**Pull request merged**